shared/util.go

```diff
@@ -33,6 +33,10 @@ const BetaLabel = "beta"
 // i.e. run from the master branch.
 const MasterLabel = "master"
 
+// WithoutPatchLabel is the label for running just the affected tests on a PR
+// but without the patch.
+const WithoutPatchLabel = "without_patch"
```

Comment:
So... I did say that eventually `master` might stop implying a full run, and this is that day.
I'm wondering if labelling the runs that aren't just the affected tests as `full` is a better route for our default homepage.

Alternatively (or even in conjunction with `full`) we can pick a more succinct name for this label. How about we go the science route, and label them as `control`? Or `baseline`? `normal`? `prior`? 
... I think `baseline` might be best, since we use that term with the same semantic meaning in other contexts.

Reply:
@foolip suggested `base` (without_patch) and `head` (with patch) offline. They work IMHO, WDYT?

Reply:
Then I threw `pr_base` and `pr_head` into the mix. Either is fine, only possible misreading is that `head` could mean HEAD, i.e. master.

Reply:
Back to your first comment, Luke. I understand `master` eventually won't be full runs in the future, but this PR doesn't change the current status that master runs are always full runs, right?

I like pr_base and pr_head, for the overuse of the term "head" itself.